scripts/booksToHTML.js

```diff
@@ -0,0 +1,55 @@
+const list = document.getElementById('list');
+/* eslint-disable no-undef */
+const toast = Swal.mixin({
```

Comment:
- [ ] Let's try to not disable Linter checks in our codebase as they are there to help us write clean, professional code. Kindly fix the problem so you have a professional codebase with no linter issue 👍🏽